src/main/java/org/sunbird/workflow/config/Configuration.java

```diff
@@ -381,4 +388,12 @@ public String getUserRegistrationServiceConfigPath() {
     public void setUserRegistrationServiceConfigPath(String userRegistrationServiceConfigPath) {
         this.userRegistrationServiceConfigPath = userRegistrationServiceConfigPath;
     }
+
+    public String getLmsSystemSettingsVerifiedProfileFieldsPath() {
+        return verifiedProfileFieldsPath;
+    }
+
+    public void setLmsSystemSettingsVerifiedProfileFieldsPath(String verifiedProfileFieldsPath) {
+        this.verifiedProfileFieldsPath = verifiedProfileFieldsPath;
+    }
```

Comment:
Keep the method name similar to variable name... 
just use the IDE's inbuilt options to generate getter setter -- do not write your own...